noir_stdlib/src/lib.nr

```diff
@@ -152,6 +147,42 @@ pub fn wrapping_mul128(x: Field, y: Field) -> Field {
 #[builtin(as_witness)]
 pub fn as_witness(x: Field) {}
 
+trait Wapping {
```

Comment:
Let's move these away from the crate root, they'd fit better somewhere in the `ops` module.

Reply:
Done